reportService/reportsServer.js

```diff
@@ -74,6 +74,12 @@ const MIN_TOP_MARGIN_PX = 40;
   const markdownArtifactsServerAddress = process.argv[15] || '';
   let browser;
 
+  console.table ? console.table({
```

Comment:
Please remove console outputs from code.

Reply:
we do use console outputs in the rest of the file. i had added `process.env.NODE_ENV` check to make sure it will run only at development envs